### PR TITLE
Remove redundant semicolons from enums

### PR DESCRIPTION
**Pull request by mureinik**
While enums allow ending the member list in a semicolon(;), it's
redundant. Some enums in the codebase use a semicolon in the end, and
some do not.

This patch standardizes the codebase's enum and cleans up the code by
removing these semicolons.

### CODE REVIEW

File: src/test/java/org/apache/commons/lang3/EnumUtilsTest.java [hunk 1]
```diff
@@ -418,10 +418,10 @@ public void test_processBitVectors_longClass() {
...
     A32, A33, A34, A35, A36, A37, A38, A39, A40, A41, A42, A43, A44, A45, A46, A47,
-    A48, A49, A50, A51, A52, A53, A54, A55, A56, A57, A58, A59, A60, A61, A62, A63;
+    A48, A49, A50, A51, A52, A53, A54, A55, A56, A57, A58, A59, A60, A61, A62, A63
 }
 enum TooMany {
     A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q,R,S,T,U,V,W,X,Y,Z,
     A1,B1,C1,D1,E1,F1,G1,H1,I1,J1,K1,L1,M1,N1,O1,P1,Q1,R1,S1,T1,U1,V1,W1,X1,Y1,Z1,
-    A2,B2,C2,D2,E2,F2,G2,H2,I2,J2,K2,L2,M2;
+    A2,B2,C2,D2,E2,F2,G2,H2,I2,J2,K2,L2,M2
 }
```
Comment (Abrasha) [inline src/test/java/org/apache/commons/lang3/EnumUtilsTest.java]: Should there be spaces between all values?

Comment (mureinik) [inline src/test/java/org/apache/commons/lang3/EnumUtilsTest.java]: I kept the style the code had. If we collectively decide that spaces would make the code better, I'd gladly add them.

Comment (PascalSchumacher) [inline src/test/java/org/apache/commons/lang3/EnumUtilsTest.java]: I guess there should, but let's keep this pull request focused one one thing. :)

**Comment by PascalSchumacher** [comment]
Thanks!

File: src/test/java/org/apache/commons/lang3/EnumUtilsTest.java (hunk 1 above)
Comment (Abrasha) [inline src/test/java/org/apache/commons/lang3/EnumUtilsTest.java]: @PascalSchumacher I will see if we need to create separate PR with enum spacing.

Comment (Abrasha) [inline src/test/java/org/apache/commons/lang3/EnumUtilsTest.java]: @PascalSchumacher created [PR #241](https://github.com/apache/commons-lang/pull/241) with minor changes of enum spacing

**Pull request closed without merging**